### PR TITLE
Create mondes-en-développement.csl

### PR DESCRIPTION
**Pull request by sbibauw**
"Mondes en développement" is a French-speaking journal in Development Studies, published by De Boeck. Instructions for authors (documentation links) describe the required bibliographic style in a very basic way (showing only a few examples), so this stylesheet is quite limited, and implementation of some details have been extrapolated from the instructions. However, I believe it still presents an interest for future authors.
This CSL is based on revue-francaise-de-sociologie.csl, by Milan Bouchet-Valet.

### CODE REVIEW

**Comment by rmzelle** [comment]
@sbibauw, thanks, but it looks like you didn't actually upload your style. Could you try again? (use the "New file" or "Upload files" button at https://github.com/sbibauw/styles )

**Comment by sbibauw** [comment]
@rmzelle Thanks. Sorry about that! It should be fine now.

File: mondes-en-developpement.csl [hunk 1]
```diff
@@ -0,0 +1,272 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never" page-range-format="expanded" demote-non-dropping-particle="never" default-locale="fr-FR">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Mondes en développement</title>
```
Comment (rmzelle) [inline mondes-en-developpement.csl]: Please change the title to `<title>Mondes en développement (French)</title>`.

**Comment by rmzelle** [comment]
Thanks!

**Comment by sbibauw** [comment]
Thank you very much!

**Pull request merged**